Shadowlands/TazaveshTheVeiledMarket/Soleah.lua

```diff
@@ -5,7 +5,10 @@
 
 local mod, CL = BigWigs:NewBoss("So'leah", 2441, 2455)
 if not mod then return end
-mod:RegisterEnableMob(177269) -- So'leah
+mod:RegisterEnableMob(
+	177269, -- So'leah
+	177716 -- So' Cartel Assassin
```

Comment:
What's the reason for this?

Reply:
I thought all mobs that the module has spells for should be listed here, the Assassins are boss summons. Is there a reason they shouldn't be included?

Reply:
@ntowle Nah, that's just for Trash. When it comes to bosses, it's really only the mobs that are in the boss health frames.